File: Client/src/content/life_1_tests.py
def RunTests() :
    fn = neighbour_cells.__name__
    def test(cell, expected) :
        n = neighbour_cells(cell)
        msg = fail +f"{fn}({cell})."
        assert_true(fn, cell, len(n) == 8, msg + f" Expected: 8 elements Actual: {len(n)}")
        for val in expected :
            assert_true(fn, cell, val in n, msg + f" List does not contain: {val}")
        
    test(30, [ 9, 10, 11, 29, 31, 49, 50, 51 ])
    all_tests_passed(fn)

File: Client/src/content/life_1_tests.cs
static void RunTests()
{
    try
    {
        TestNeighbourCells(30, new List<int> { 9, 10, 11, 29, 31, 49, 50, 51 });
        AllTestsPassed();
    }
    catch (TestFailure) { }
}

private static void TestNeighbourCells(int cell, List<int> expected)
{
    var n = NeighbourCells(cell);
    string fn = nameof(NeighbourCells);
    var msg = fail + $"{fn}({cell}).";
    AssertTrue(fn, cell.ToString(), n.Count == 8, msg + $" Expected: 8 elements Actual: {n.Count}");
    foreach (int val in expected)
    {
        AssertTrue(fn, cell.ToString(), n.Contains(val), msg + $" List does not contain: {val}");
    }
}